### DISCUSSION TITLE
Erasure-Code-擦除码-3-极限篇 - OpenACID Blog

### DISCUSSION THREAD

**Discussion by giscus** [openacid/openacid.github.io#32] [General]
# Erasure-Code-擦除码-3-极限篇 - OpenACID Blog

没有人比我更懂EC(特辣脯口气): 从指令集和算法两个方向对擦除码实现深度优化, 硬核但不烧脑

https://blog.openacid.com/storage/ec-3/

**Comment by hzxGoForward** [comment]
这三篇关于 EC 的原理,讲的太棒了!

**Reply by drmingdrmer** [reply]
˙∆˙谢谢喜欢

**Comment by NUABO** [comment]
对于小io覆盖写在ec中使用，大佬有什么建议吗

**Reply by drmingdrmer** [reply]
ec对小io不友好, 想办法攒一堆批量写会好点.

**Reply by NUABO** [reply]
感谢(:

**Reply by NUABO** [reply]
还想请问一下，开源代码有小io聚合类似的实现吗

**Comment by daxiguafan** [comment]
"这里p是单块数据丢失的风险,一般选择磁盘的日损坏率: 大约是0.0001. p一般很小所以近似数据丢失风险就只看第1项:" 看这个表达式，如果m不变，似乎K越大，风险值越小？是近似导致的吗

**Reply by drmingdrmer** [reply]
k越大风险越大啊
